examples/realtime-mc.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: tuned.openshift.io/v1
+kind: Tuned
+metadata:
+  name: openshift-realtime
+  namespace: openshift-cluster-node-tuning-operator
+spec:
+  profile:
+  - data: |
+      [main]
+      summary=Custom OpenShift realtime profile
+      include=openshift-node,realtime
+      [variables]
+      # isolated_cores take a list of ranges; e.g. isolated_cores=2,4-7
+      isolated_cores=1
```

Comment:
This will directly utilize https://github.com/redhat-performance/tuned/blob/master/profiles/cpu-partitioning/tuned.conf ?

Reply:
> This will directly utilize https://github.com/redhat-performance/tuned/blob/master/profiles/realtime/tuned.conf?

Yes, see the patch in `/assets/patches/060-realtime.diff`.